src/main/java/chessmaster/game/Game.java

```diff
@@ -74,6 +79,11 @@ private Command getUserCommand() throws ChessMasterException {
         return command;
     }
 
+    private void handleShowMoves() throws ChessMasterException {
```

Comment:
This is not needed since all execution of showMoveCommand is done in the command class itself